### PR TITLE
azure-endpoint-revisit

### PR DESCRIPTION
**Pull request by resmaa**
Added API Allocation Load Balancer to Azure Cluster creation wizard

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1011#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 94f206a0c756253a881ef0d19a1c9888342a23a6 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1011/librarium_build/1598002699868049408) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 94f206a0c756253a881ef0d19a1c9888342a23a6 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1011/librarium_build/1598002699863855104) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04-clusters/01-public-cloud/03-azure.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure.md:59]:
```suggestion
For existing cloud account go to **Edit** and toggle the **Connect Private Cloud Gateway** option to select the created Gateway from the drop-down menu.

```

File: content/docs/04-clusters/01-public-cloud/03-azure.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure.md:61]: I don't understand what is being said here. Also, we should address the user with "you" and not use a third-person voice such as "if the user."  But this paragraph needs work as it's confusing to read.

File: content/docs/04-clusters/01-public-cloud/03-azure.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure.md:79]: The last sentence is a prerequisite, so if this was structured into a How-to this would be addressed.

File: content/docs/04-clusters/01-public-cloud/03-azure.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure.md:82]:
```suggestion
2. Select the **Cluster Profile** created for the Azure environment. The profile definition will be used as the cluster construction template. Click on **Next**.

```

File: content/docs/04-clusters/01-public-cloud/03-azure.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure.md:85]:
```suggestion
3. Review and override pack parameters as desired. By default, parameters for all packs are set with values defined in the Cluster Profile. Click **Next**.

```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Resma, let's take this as an opportunity to improve on this page.  Take a look at the page navigation (see the image below). There is way too much happening on one documentation page. Those are all How-to topics. We need to clean this up as we did with App Profile. Use App Profile as an example of how to restructure this section.  This page should be the overview that provides links to the other pages related to Azure. 

CleanShot 2022-11-30 at 13 25 21

File: content/docs/04-clusters/01-public-cloud/03-azure.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/03-azure.md:140]:
```suggestion
You can add new worker pools to customize certain worker nodes to run specialized workloads. For example, the default worker pool may be configured with the <i>Standard_D2_v2</i> instance types for general-purpose workloads and another worker pool with instance type <i>Standard_NC12s_v3</i> can be configured to run GPU workloads.

```

**Review by ritawatson — CHANGES_REQUESTED** [review]

**Comment by resmaa** [comment]
> Resma, let's take this as an opportunity to improve on this page. Take a look at the page navigation (see the image below). There is way too much happening on one documentation page. Those are all How-to topics. We need to clean this up as we did with App Profile. Use App Profile as an example of how to restructure this section. This page should be the overview that provides links to the other pages related to Azure.
> 
> CleanShot 2022-11-30 at 13 25 21

Sure we can

**Review by karl-cardenas-coding — APPROVED** [review]

**Pull request merged**
